architectures/PGD-Always-ON/upgrade_major_3to5.yml

```diff
@@ -241,6 +241,11 @@
       vars:
         upgrade_from: 3
     - include_role: name=postgres/user
+    - include_role:
+        name: postgres/update
+        tasks_from: preconfig.yml
+      vars:
+        upgrade_from: 3
```

Comment:
Since we are not doing any update or preconfig change anymore, should we keep postgres/update/preconfig.yml? Perhaps the couple of tasks in preconfig.yml can be moved here?

Reply:
I would keep in it a separate file since we might want to use it in the other upgrade scenario 4 to 5 to use the correct version there too in config.